web/src/main/java/nl/tehdreamteam/se42/web/command/impl/HelpCommand.java

```diff
@@ -0,0 +1,34 @@
+package nl.tehdreamteam.se42.web.command.impl;
+
+import nl.tehdreamteam.se42.web.command.Command;
+import nl.tehdreamteam.se42.web.command.CommandHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HelpCommand extends Command {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Document this public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)